app/core/api.js

```diff
@@ -775,29 +775,43 @@ api.getUploadUrl = function() {
 api.metrics = {};
 
 api.metrics.track = function(eventName, properties, cb) {
-  if (typeof window._paq !== 'undefined') {
-    // Using Matomo Tracker
-    api.log(`Matomo trackEvent ${eventName}:`, properties);
-    if (eventName === 'CookieConsent') {
-      window._paq.push(['setConsentGiven', properties]);
-    } else if (eventName === 'setCustomUrl') {
-      window._paq.push(['setCustomUrl', properties]);
-    } else if (eventName === 'setUserId') {
-      window._paq.push(['setUserId', properties]);
-    } else if (eventName === 'resetUserId') {
-      window._paq.push(['resetUserId']);
-    } else if (eventName === 'setDocumentTitle' && typeof properties === 'string') {
-      window._paq.push(['setDocumentTitle', properties]);
-    } else if (typeof properties === 'undefined') {
-      window._paq.push(['trackEvent', eventName]);
+  const metricsService = _.get(config, 'METRICS_SERVICE', 'highwater');
```

Comment:
It was to keep blip as much as possible compatible with the original blip on default config.
Like we always do.
I won't be stubborn on it. It would be me only, all the highwater code will have been removed.